src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -203,6 +203,7 @@ class IOUAmountPage extends React.Component {
                                 ref={el => this.textInput = el}
                                 value={this.state.amount}
                                 placeholder="0"
+                                keyboardType="decimal-pad"
```

Comment:
use CONST instead.

Reply:
@parasharrajat, we didn't have a variable for decimal pad in CONST.js so I have added one and used it.